definitions/tools/star_fusion_detect.cwl

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env cwl-runner
+
+
+class: CommandLineTool
+cwlVersion: "v1.0"
+label: "STAR-Fusion identify candidate fusion transcript"
+
+baseCommand: ["/usr/local/src/STAR-Fusion/STAR-Fusion"]
+requirements:
+    - class: ShellCommandRequirement 
+    - class: ResourceRequirement
+      ramMin: 64000
+      coresMin: 10 
+    - class: DockerRequirement
+      dockerPull: trinityctat/starfusion:1.8.0
+
+arguments: [
+    "--CPU", $(runtime.cores)
+]
+
+
+inputs:
+  star_fusion_genome_dir:
+    type: Directory
+    inputBinding:
+        position: 2
+        prefix: '--genome_lib_dir'
+    doc: '
+    fusion genome dir
+    '
+
+  junction_file:
```

Comment:
Can you add a short doc line here explaining what this is?

Reply:
Added doc